azure-kusto-ingest/azure/kusto/ingest/_streaming_ingest_client.py

```diff
@@ -97,12 +97,16 @@ def _ingest(self, stream, size, ingestion_properties, content_encoding=None):
         ):
             raise KustoMissingMappingReferenceError()
 
+        client_request_properties = ClientRequestProperties()
```

Comment:
I find it strange that you create the client request properties.
This way the customer cannot provide client activity id.